scss/dijit/layout/_tabContainer.scss

```diff
@@ -200,9 +200,7 @@
     &ListContainer-top,
     &ListContainer-bottom {
       @include box-sizing("content-box");
-      border-width: $tab-list-container-border-width;
-      border-style: $tab-list-container-border-style;
-      border-color: $tab-list-container-border-color;
+      box-shadow: -2px 2px 2px #c5c5c5;
```

Comment:
We have a var for this: https://github.com/dotCMS/dotcms-dojo-theme/blob/master/scss/_dijit-theme-dotcms.scss#L7